### PR TITLE
Fix flare shell not applying firestacks and makes it embed

### PR DESCRIPTION
**Pull request by Arendian**
## About the PR

Flare shells currently don't ignite targets while they are supposed to, this PR makes them apply their firestack again. Flare shell projectiles now also embed and deal a tiny bit of heat damage on impact. 
Removing the embedded shell will delete it, giving people an option to destroy a flare if they think it's too bright. 
Currently on master you have to wait the full 4 minutes for it to despawn by itself.

## Why / Balance

Getting hit by a flaming hot flare should probably light you on fire a little bit, current firestacks will do 13 damage if someone starts rolling immediately, and around 21 damage when someone doesn't drop and roll at all. Getting hit with multiple shells in quick succession increases this damage exponentially, but rolling once removes around 3 shells worth of fire stacks so it should still be fine.

Making it embed allows the shell to act like a projectile without bouncing around the room the moment they hit a wall at an angle. Making it act like a projectile makes it more useful than it was in it's previous state, where it basically was a glorified emergency flare because you couldn't shoot it further than you can throw stuff.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Dygon
- tweak: Flare shells now embed into whatever they hit.
- fix: Flare shells ignite mobs again.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml [hunk 1]
```diff
@@ -137,6 +137,13 @@
           - ItemMask
         restitution: 0.3
         friction: 0.2
+    ignition:
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml:140]: image

check fails is real

**Review by TheShuEd — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml (hunk 1 above)
Comment (Arendian) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml:140]: it's added in the the PR linked in the description

Comment (Arendian) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml:140]: I'll probably just move the fix over to this PR and keep the firestack changes in the otther one

Comment (TheShuEd) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml:140]: ah, okay, I see. Then we wait.

Comment (Arendian) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml]: Should be fine to merge now

**Review by TheShuEd — APPROVED** [review]

**Pull request merged**